brain-marks/Settings/Views/SettingsView.swift

```diff
@@ -37,7 +37,15 @@ struct SettingsView<ViewModel: InfoViewModel>: View {
                             }
                         }
                     }
-                    
+                    Section(header: Text("Updates")) {
+                                          NavigationLink {
+                                            WhatsNewView(viewModel: viewModel)
+                                          } label: {
+                                              HStack {
+                                                  Text("What's New")
+                                              }
+                                          }
+                                      }
```

Comment:
Can you fix the indentation here?
The `NavigationLink` shouldn't be indented this far

Also the `HStack` isn't needed